operators/flux/0.36.0/manifests/flux.v0.36.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1556 @@
+              "name": "github-receiver",
+              "namespace": "default"
+          },
+          "spec": {
+              "type": "github",
+              "events": [
+                "ping",
+                "push"
+              ],
+              "secretRef": {
+                "name": "webhook-token"
+              },
+              "resources": [
+                {
+                    "kind": "GitRepository",
+                    "name": "webapp"
+                },
+                {
+                    "kind": "HelmRepository",
+                    "name": "webapp"
+                }
+              ]
+          }
+        },
+        {
+          "apiVersion": "image.toolkit.fluxcd.io/v1beta1",
+          "kind": "ImageRepository",
+          "metadata": {
+              "name": "podinfo",
+              "namespace": "flux-system"
+          },
+          "spec": {
+              "image": "ghcr.io/stefanprodan/podinfo",
+              "interval": "1m0s"
+          }
+        },
+        {
+          "apiVersion": "image.toolkit.fluxcd.io/v1beta1",
+          "kind": "ImagePolicy",
+          "metadata": {
+              "name": "podinfo",
+              "namespace": "flux-system"
+          },
+          "spec": {
+              "imageRepositoryRef": {
+                "name": "podinfo"
+              },
+              "policy": {
+                "semver": {
+                    "range": "5.0.x"
+                }
+              }
+          }
+        },
+        {
+          "apiVersion": "image.toolkit.fluxcd.io/v1beta1",
+          "kind": "ImageUpdateAutomation",
+          "metadata": {
+            "name": "flux-system",
+            "namespace": "flux-system"
+          },
+          "spec": {
+            "interval": "1m0s",
+            "sourceRef": {
+              "kind": "GitRepository",
+              "name": "flux-system"
+            },
+            "git": {
+              "checkout": {
+                "ref": {
+                  "branch": "main"
+                }
+              },
+              "commit": {
+                "author": {
+                  "email": "fluxcdbot@users.noreply.github.com",
+                  "name": "fluxcdbot"
+                },
+                "messageTemplate": "{{range .Updated.Images}}{{println .}}{{end}}"
+              },
+              "push": {
+                "branch": "main"
+              }
+            },
+            "update": {
+              "path": "./clusters/my-cluster",
+              "strategy": "Setters"
+            }
+          }
+        }
+      ]
+    capabilities: Seamless Upgrades
+    categories: Integration & Delivery
+    certified: "false"
+    description: Flux is a Continuous Delivery solution for Kubernetes.
+    support: The Flux community
+    operatorframework.io/suggested-namespace: flux-system
+    containerImage: quay.io/redhat-isv-containers/62b4cc4ce987873deb78d87d@sha256:fb607923a6dc9a19bdcf6327f1bedbc9016e4c46fcd5511aa30eb714d9b3e9fc
```

Comment:
@paulofrazao I think there is some confusing, quay is the `inbound` registry where you push the images to, like how it used to be `scan`. The images are still served to customers from `registry.connect.redhat.com` can you do a diff between the latest certified version and only change the `sha's` of the images here?

Reply:
Hi Adam, thanks, yes confusing, partner portal shows quay.io. I just changed it back to registry.connect.redhat.com pointing to new sha256 keys for new images.